### PR TITLE
PRM-2793 make PoC achievement date optional

### PR DESCRIPTION
**Pull request by RoheelArshad**

### CODE REVIEW

File: test/models/auditing/UserHasPenaltyAuditModelSpec.scala [hunk 1]
```diff
@@ -369,7 +369,7 @@ class UserHasPenaltyAuditModelSpec extends SpecBase with LogCapturing {
         inactivePenaltyPoints = 2,
         regimeThreshold = 5,
         penaltyChargeAmount = 200,
-        PoCAchievementDate = LocalDate.of(2022, 1, 1)
+        PoCAchievementDate = Some(LocalDate.of(2022, 1, 1))
```
Comment (LewisMitchell) [inline test/models/auditing/UserHasPenaltyAuditModelSpec.scala:372]: Apparently, PoCAchievementDate is None when the `activePenaltyPoints = 0`, could you set this to None?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

**Comment by LewisMitchell** [comment]
Could you also rebase, please?

File: test/models/auditing/UserHasPenaltyAuditModelSpec.scala (hunk 1 above)
Comment (RoheelArshad) [reply test/models/auditing/UserHasPenaltyAuditModelSpec.scala]: changed

**Comment by RoheelArshad** [comment]
> Could you also rebase, please?

Done

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**